core/templates/domain/learner_dashboard/learner-dashboard-backend-api.service.spec.ts

```diff
@@ -353,15 +353,15 @@ describe('Learner Dashboard Backend API Service', () => {
     let successHandler = jasmine.createSpy('success');
     let failHandler = jasmine.createSpy('fail');
 
-    let updatedStatus = null;
-    let updatedSubject = null;
+    let updatedStatus = true;
+    let updatedSubject = 'Updated Subject';
     let text = 'Sending message';
     let url = '/threadhandler/exploration.4.Wfafsafd';
     let payload = {
       updated_status: updatedStatus,
       updated_subject: updatedSubject,
       text: text
-    };
+    } as AddMessagePayload;
```

Comment:
why not: `let payload: AddMessagePayload = {` instead of `as AddMessagePayload`.

Reply:
done